dev-util/coccinelle/coccinelle-1.0.7.ebuild

```diff
@@ -0,0 +1,127 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python2_7 )
+
+inherit multilib eutils python-single-r1 bash-completion-r1 elisp-common autotools
+
+MY_P="${P/_/-}"
+DESCRIPTION="Program matching and transformation engine"
+HOMEPAGE="http://coccinelle.lip6.fr/"
+SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc emacs ocaml +ocamlopt pcre python test vim-syntax"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# ocaml enables ocaml scripting (uses findlib)
+COMMON_DEPEND=">=dev-lang/ocaml-3.12:=[ocamlopt?]
+	dev-ml/sexplib:=[ocamlopt(+)?]
+	dev-ml/menhir:=[ocamlopt?]
+	dev-ml/camlp4:=[ocamlopt?]
+	dev-ml/parmap:=[ocamlopt?]
+	emacs? ( virtual/emacs )
+	ocaml? ( dev-ml/findlib:= )
+	pcre? ( dev-ml/pcre-ocaml:=[ocamlopt(+)?] )
+	python? ( ${PYTHON_DEPS} )"
+
+RDEPEND="${COMMON_DEPEND}
+	vim-syntax? ( || ( app-editors/vim app-editors/gvim ) )"
+
+# dev-texlive/texlive-fontsextra contains 'ifsym.sty'
+DEPEND="${COMMON_DEPEND}
+	virtual/pkgconfig
+	doc? (
+		virtual/latex-base
+		dev-texlive/texlive-latexextra
+		dev-texlive/texlive-fontsextra
+	)"
+
+DOCS=( authors.txt bugs.txt changes.txt credits.txt readme.txt )
+
+RESTRICT=strip
+S="${WORKDIR}/${MY_P}"
+
+SITEFILE=50coccinelle-gentoo.el
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	default
+
+	if use python ; then
+		# fix python install location
+		sed -e "s:\$(LIBDIR)/python:$(python_get_sitedir):" \
+			-i Makefile || die
+	fi
+
+	sed -e '/have spatch/d' -i scripts/spatch.bash_completion || die
+
+	AT_M4DIR=setup eaclocal
+	eautoconf
+	cd bundles/stdcompat/stdcompat-current && eaclocal && eautoconf && eautomake
```

Comment:
Please split these commands into new lines, also you need to append the first `cd ...` with `|| die`

Rest looks good!

Reply:
Removed this line. Works without it.